Gemfile

```diff
@@ -14,6 +14,7 @@ gem 'uglifier', '>= 1.3.0'
 gem 'coffee-rails', '~> 4.2'
 gem 'jbuilder', '~> 2.5'
 gem 'bcrypt'
+gem 'table_print'
```

Comment:
Asked Mike for permission to use this gem. When in `rails c`, if you write `tp` before your command then the return value is in a nice little table. It didn't end up helping me solve the riddle I was hoping to solve, but it is cool and useful.

Reply:
love it

Reply:
This is so helpful, gunna use this now too!